run.py

```diff
@@ -82,12 +82,24 @@ def get_watson_online_store():
         cloudant_password = os.environ.get("CLOUDANT_PASSWORD")
         cloudant_url = os.environ.get("CLOUDANT_URL")
         cloudant_db_name = os.environ.get("CLOUDANT_DB_NAME")
-
-        # TODO: It looks like we'll want to make discovery required too.
         discovery_username = os.environ.get('DISCOVERY_USERNAME')
-        discovery_password = os.environ.get('DISCOVERY_PASSWORD')
         discovery_environment_id = os.environ.get('DISCOVERY_ENVIRONMENT_ID')
-        discovery_collection_id = os.environ.get('DISCOVERY_COLLECTION_ID')
+        discovery_password = os.environ.get('DISCOVERY_PASSWORD')
+        discovery_source = os.environ.get('DISCOVERY_DATA_SOURCE')
+        discovery_collection_id = None
+        discovery_score_filter = 1.0
+        if discovery_source:
+            discovery_collection_id = \
+                os.environ.get(discovery_source + '_DISCO_COLLECTION_ID')
+
+            try:
+                discovery_score_filter = float(os.environ.get(
+                    discovery_source + "_DISCO_SCORE_FILTER", 0))
```

Comment:
I'm removing all of the disco checks other than uname and password. Was hoping to catch everything here, but still need to verify again in online_store. So, will just do it in online_store.